man/systemd.exec.xml

```diff
@@ -1025,10 +1025,12 @@ CapabilityBoundingSet=~CAP_B CAP_C</programlisting>
       <varlistentry>
         <term><varname>CPUSchedulingResetOnFork=</varname></term>
 
-        <listitem><para>Takes a boolean argument. If true, elevated CPU scheduling priorities and policies will be
-        reset when the executed processes fork, and can hence not leak into child processes. See
-        <citerefentry><refentrytitle>sched_setscheduler</refentrytitle><manvolnum>2</manvolnum></citerefentry> for
-        details. Defaults to false.</para></listitem>
+        <listitem><para>Takes a boolean argument. If true, elevated CPU scheduling priorities and policies
+        will be reset when the executed processes call
+        <citerefentry project='man-pages'><refentrytitle>fork</refentrytitle><manvolnum>2</manvolnum></citerefentry>,
+        and can hence not leak into child processes. See
+        <citerefentry><refentrytitle>sched_setscheduler</refentrytitle><manvolnum>2</manvolnum></citerefentry>
```

Comment:
should get `project='manpages'` too i guess

Reply:
I'll add that in a later patch. I don't think it makes sense to respin the CI just for that.